server/pulp/server/agent/direct/services.py

```diff
@@ -229,7 +229,7 @@ def failed(self, reply):
             finished = dateutils.format_iso8601_datetime(now)
 
         TaskStatus.objects(task_id=task_id).update_one(set__finish_time=finished,
-                                                       set__state=constants.CALL_ERRORED_STATE,
+                                                       set__state=constants.CALL_FAILED_STATE,
                                                        set__traceback=traceback)
 
         if action == 'bind':
```

Comment:
I really need to strip out the agent stuff soon so we don't feel the need to make these changes.